src/components/graph-view.js

```diff
@@ -692,20 +692,22 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
       const deltaX = position.x - node.x;
       const deltaY = position.y - node.y;
 
-      const nodesToMove = selectedNodes.find(n => n[nodeKey] === node[nodeKey])
+      const isSelected = selectedNodes.find(n => n[nodeKey] === node[nodeKey]);
+
+      const nodesToMove = isSelected
         ? selectedNodes
         : selectedNodes.concat([node]);
 
       GraphUtils.yieldingLoop(nodesToMove.length, 50, i => {
         const node = nodesToMove[i];
         const nodeMapNode = this.getNodeById(node[nodeKey]);
 
-        // node moved
+        // move node before rendering edges
         node.x += deltaX;
         node.y += deltaY;
 
         this.renderConnectedEdgesFromNode(nodeMapNode, true);
```

Comment:
what would happen if we just removed this instead of passing the flag into `asyncRenderNode`?

Reply:
I tried this at first but it causes the edge rendering to look more out-of-sync because of the order in which the animation frames are requested -- the edges don't smoothly move with the nodes but instead lag one or two frames behind them.